packages/angular/cli/models/command.ts

```diff
@@ -115,6 +115,9 @@ export abstract class Command<T extends BaseCommandOptions = BaseCommandOptions>
           if (o.description) {
             this.logger.info(formatDescription(o.description));
           }
+          if (o.default !== undefined) {
+            this.logger.info('    Default: ' + o.default);
```

Comment:
Can we add this on the same line as descriptions please? Similar to what is posted in the original request?

Thanks.

Reply:
Just made this change.